### PR TITLE
[plsql] Fix in Comment statement

### PR DESCRIPTION
**Pull request by szyman23**
Before submitting a PR, please check that:
 - [ ] The PR is submitted against `master`. The PMD team will merge back to support branches as needed.
 - [ ] `./mvnw clean verify` passes. This will [build](https://github.com/pmd/pmd/blob/master/BUILDING.md) and test PMD, execute PMD and checkstyle rules. [Check this for more info](https://github.com/pmd/pmd/blob/master/CONTRIBUTING.md#code-style)

**PR Description:**
Small fix in Comment statement with test case.
Ending comment without ";", but with "/" is accepted by SQLPlus
Also removed not needed productions (CursorSpecification, ...) and reformat some code.

### CODE REVIEW

**Comment by pmd-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">No java rules are changed!</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: pmd-plsql/etc/grammar/PldocAST.jjt [hunk 1]
```diff
@@ -1083,19 +1079,13 @@ void Skip2NextTokenOccurrence(int target) :
     */
     specToken = nextToken.specialToken;
     if (null!= specToken)
-    {
```
Comment (adangel) [inline pmd-plsql/etc/grammar/PldocAST.jjt]: I'd rather keep these braces. What happens if someone later wants to add a second statement and forgets to put it into the braces?

File: pmd-plsql/etc/grammar/PldocAST.jjt [hunk 2]
```diff
@@ -473,8 +473,6 @@ ASTDeclarativeUnit DeclarativeUnit()  :
 		ProgramUnit()  |
 		LOOKAHEAD(4)
 		VariableOrConstantDeclaration() |
-		LOOKAHEAD(2)
-		CursorSpecification() |
```
Comment (adangel) [inline pmd-plsql/etc/grammar/PldocAST.jjt]: Removing CursorSpecification actually breaks compatibility, because now the Cursor is parsed as CursorBody

Comment (szyman23) [reply pmd-plsql/etc/grammar/PldocAST.jjt]: You're right, thanks.

**Pull request merged**